torchaudio_contrib/functional.py

```diff
@@ -212,20 +228,25 @@ def phase_vocoder(complex_specgrams, rate, phase_advance):
         >>> x.shape # with 231 == round(300 / 1.3)
         torch.Size([16, 1, 1025, 231, 2])
     """
+    ndim = complex_specgrams.dim()
+    time_slice = [slice(None)] * (ndim - 2)
```

Comment:
Using `slice` so as to not add a bunch of if statements to deal with batch vs. no batch...